lib/configService.js

```diff
@@ -68,6 +68,7 @@ function processEnvironmentVariables() {
             'IOTA_MQTT_RETRIES',
             'IOTA_MQTT_RETRY_TIME',
             'IOTA_MQTT_KEEPALIVE',
+            'IOTA_MQTT_AVOID_LEADING_SLASHES',
```

Comment:
Both the new mqtt.avoidLeadingSlash configuration parameter and associated env var should be included in documentation in https://github.com/telefonicaid/iotagent-json/blob/master/docs/installationguide.md#configuration

Reply:
Sure. These kind of aspects are what I meant when I asked about a standard procedure about PRs.

Reply:
Done.